### PR TITLE
Fix bedrock notebook retrieve call and add llamahub doc improvement

### PR DESCRIPTION
**Pull request by rozerarenu**
# Description
Fix bedrock retriever notebook retrieve call resulted with this [change](https://github.com/run-llama/llama_index/pull/12910/files) and add minor llamahub documentation improvement.

Fixes # (issue)
N/A

## New Package?

Did I fill in the `tool.llamahub` section in the `pyproject.toml` and provide a detailed README.md for my new integration or package?

- [ ] Yes
- [X] No

## Version Bump?

Did I bump the version in the `pyproject.toml` file of the package I am updating? (Except for the `llama-index-core` package)

- [ ] Yes
- [X] No

## Type of Change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [X] Added new notebook (that tests end-to-end)
- [ ] I stared at the code and made sure it makes sense

## Suggested Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: llama-index-integrations/retrievers/llama-index-retrievers-bedrock/README.md [hunk 1]
```diff
@@ -10,10 +10,19 @@
...
-Explore the retriever using Notebook present at:
+```
+pip install llama-index-retrievers-bedrock
+```
+
+## Usage
 
 ```
-docs/docs/examples/retrievers/bedrock_retriever.ipynb
+from llama_index.retrievers.bedrock import AmazonKnowledgeBasesRetriever
```
Comment (nerdai) [inline llama-index-integrations/retrievers/llama-index-retrievers-bedrock/README.md:22]: Maybe useful to show how the obj can be used in addition to being imported?

**Review by nerdai — COMMENTED** [review]
Thankss @rozerarenu! Left a comment for you. :)

**Review by logan-markewich — APPROVED** [review]

File: llama-index-integrations/retrievers/llama-index-retrievers-bedrock/README.md (hunk 1 above)
Comment (logan-markewich) [reply llama-index-integrations/retrievers/llama-index-retrievers-bedrock/README.md:22]: I just copied the example from the notebook lol

Comment (nerdai) [reply llama-index-integrations/retrievers/llama-index-retrievers-bedrock/README.md:22]: lol perfect

**Pull request merged**